atlas-fastq-provider-functions.sh

```diff
@@ -138,6 +149,29 @@ validate_ena_ssh_path() {
     fi
 }
 
+
+# Use AWS S3 to check if the file is in ENA. Note that the '-n' is important,
+# because any 'while read' loops calling this script use STDIN, which SSH will
+# consume otherwise.
+
+validate_ena_fire_path() {
+    local enaFile=$1
+    
+    check_ena_s3
```

Comment:
```suggestion
    check_ena_s3_profile
```

Reply:
Done in 27c8b346819a046c361956a4e6c96d5d7a70094d